modules/mobile/app/controllers/mobile/discovery_controller.rb

```diff
@@ -6,10 +6,171 @@ module Mobile
+        },
+        endpoints: {
+          appeal_details: {
+            url: '/appeal/:id',
+            method: 'GET'
+          },
+          appointments: {
+            url: '/appointments',
+            method: 'GET'
+          },
+          claims_overview: {
+            url: '/claims-and-appeals-overview',
+            method: 'GET'
+          },
+          claim_details: {
+            url: '/claim/:id',
+            method: 'GET'
+          },
+          upload_claim_documents: {
+            url: '/claim/:id/documents',
+            method: 'POST'
+          },
+          available_letters: {
+            url: '/letters',
+            method: 'GET'
+          },
+          letters_beneficiary: {
+            url: '/letters/beneficiary',
+            method: 'GET'
+          },
+          letters_download: {
+            url: '/letters/:type/download',
+            method: 'POST'
+          },
+          service_history: {
+            url: '/military-service-history',
+            method: 'GET'
+          },
+          get_payment_info: {
+            url: '/payment-information/benefits',
+            method: 'GET'
+          },
+          update_payment_info: {
+            url: '/payment-information/benefits',
+            method: 'PUT'
+          },
+          user: {
+            url: '/user',
+            method: 'GET'
+          },
+          logout: {
+            url: '/user/logout',
+            method: 'GET'
+          },
+          create_user_addresses: {
+            url: '/user/addresses',
+            method: 'POST'
+          },
+          update_user_addresses: {
+            url: '/user/addresses',
+            method: 'PUT'
+          },
+          user_address_validation: {
+            url: '/user/addresses/validate',
+            method: 'POST'
+          },
+          create_user_email: {
+            url: '/user/emails',
+            method: 'POST'
+          },
+          update_user_email: {
+            url: '/user/emails',
+            method: 'PUT'
+          },
+          create_user_phone: {
+            url: '/user/phones',
+            method: 'POST'
+          },
+          update_user_phone: {
+            url: '/user/phones',
+            method: 'PUT'
+          }
+        },
+        display_message: '',
+        app_access: true
+      }
+    }.freeze
+
+    OAUTH_ENV_MAP = {
+      dev: 'https://sqa.fed.eauth.va.gov/oauthe/sps/oauth/oauth20/',
+      staging: 'https://int.fed.eauth.va.gov/oauthe/sps/oauth/oauth20/',
+      prod: 'https://fed.eauth.va.gov/oauthe/sps/oauth/oauth20/'
+    }.freeze
+
+    API_ROOT_ENV_MAP = {
+      dev: 'https://staging-api.va.gov/mobile',
+      staging: 'https://staging-api.va.gov/mobile',
+      prod: 'https://api.va.gov/mobile'
+    }.freeze
+
+    def welcome
       render json: { data: { attributes: { message: 'Welcome to the mobile API' } } }
     end
+
+    def index
+      response = OpenStruct.new
```

Comment:
Right now the discovery info is static but eventually will be dynamic what do you think about making this OpenStruct a model and moving all the business logic (the maps above) there?

Reply:
Sounds good to me, I'll get on that asap